pylutron/__init__.py

```diff
@@ -358,14 +358,13 @@ def _parse_button(self, keypad, component_xml):
     engraving = button_xml.get('Engraving')
     button_type = button_xml.get('ButtonType')
     direction = button_xml.get('Direction')
-    led_logic = button_xml.get('LedLogic')
-    name = button_xml.get('Name')
+    led_logic = 0 if button_xml.get('LedLogic') is None else int(button_xml.get('LedLogic'))
+    name = f"keypad {keypad.id}: Btn {component_number}"
```

Comment:
please do not rename. Since this button is attached to a keypad, it's already properly "namespaced". Let our clients decide how to combine the names together.

Reply:
I do remember you were discussing properly unique identifying lutron objects, so I thought was the right moment to do it.
I've renamed f"keypad {keypad.id}: Btn {component_number}", f"keypad {keypad.id}: Led {component_number}" and f"keypad {keypad.id}: CCI {component_number}". The keypad is f"keypad {keypad.id}".
What do you think?

Reply:
@thecynic: UUID from @cdheiser are not adding uniqueness in Homeworks. We don't have UUID in the XML.
Regarding the naming . HASS is setting the name as `return f"{self._area_name} {self._lutron_device.name}",`,
So if you have more keypads in the same room you will end up with multiple buttons with the same name. How do you distinguish them?
And the engraving is actually the real meaningful name for the button.
Also remember that in a keypad there are also leds and CCIs. So it make sense for me to have a consistent name like `keypad_intID_btn_xxx`